src/getHumanAge.test.js

```diff
@@ -7,4 +7,37 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0, if animalAge less than first', () => {
+    const result = getHumanAge(14, 14);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test(`should return 1,
+      if animalAge less than (first + second) and more than first`, () => {
+    const result = getHumanAge(23, 23);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test(`should return 2, if animalAge equal (first + second)`, () => {
+    const result = getHumanAge(27, 27);
+
+    expect(result).toEqual([2, 2]);
+  });
+
+  test(`should return correct value,
+      if animalAge more than (first + second)`, () => {
+    const result = getHumanAge(28, 28);
+
+    expect(result).toEqual([3, 2]);
+  });
+
+  test(`should return correct value, if one animalAge equal (first + second)
```

Comment:
There is a typo in the test description: 'animalAgemore' should be 'animalAge more'. Additionally, the description should clarify that one of the ages is exactly 24 and the other is more than 24.